### PR TITLE
Api schema

### PR DESCRIPTION
**Pull request by devinreuschel**
Added API schema to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,86 @@ Shipping component
...
+      "image": "url"
+    }
+```
+
+
+### Update product shipping info
+  * PATCH `/products/:id`
+
+**Path Parameters:**
+  * `id` restaurant id
```
Comment (serviolee) [inline README.md:65]: did you mean product id?

File: README.md [hunk 2]
```diff
@@ -4,3 +4,86 @@ Shipping component
...
+      "image": "url"
+    }
+```
+
+
+### Delete product
+  * DELETE `/products/:id`
+
+**Path Parameters:**
+  * `id` restaurant id
```
Comment (serviolee) [inline README.md:87]: again, did you mean product id?

File: README.md [hunk 3]
```diff
@@ -4,3 +4,86 @@ Shipping component
...
+      "image": "url"
+    }
+```
+
+
+### Update product shipping info
+  * PATCH `/products/:id`
+
+**Path Parameters:**
+  * `id` restaurant id
```
Comment (Maksyym) [inline README.md:65]: restaurant? <.<

File: README.md [hunk 4]
```diff
@@ -4,3 +4,86 @@ Shipping component
...
+      "image": "url"
+    }
+```
+
+
+### Delete product
+  * DELETE `/products/:id`
+
+**Path Parameters:**
+  * `id` restaurant id
```
Comment (Maksyym) [inline README.md:87]: restaurant? <.<

**Review by serviolee — APPROVED** [review]

**Pull request closed without merging**